src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -22,8 +22,8 @@ private void TestWindowApisUnix()
             // This tests the hardcoded implementations of these APIs on Unix.
             using (Process p = Process.GetCurrentProcess())
             {
-                Assert.True(p.Responding);
-                Assert.Equal(string.Empty, p.MainWindowTitle);
+                Assert.Throws<PlatformNotSupportedException>(() => p.Responding);
```

Comment:
I think the defaults we had are better.  There is no concept of "not responding", so true makes sense.  And there is no "window title", so it's empty.  Throwing would make sense if the app consuming these would be negatively influenced by the default values we're returning... do you have an example where these would reasonably cause a problem?